doc/contributors-guide/making-and-submitting-changes.md

```diff
@@ -8,6 +8,14 @@ Add video here
 
 Add text content
 
+### Training repo: Add your changes using the GitHub UI
+
+- Fork the training repo (Additional information: The default branch of the training repo is the main branch. So on the "Create a new fork" page, make sure to uncheck the "Copy the main branch only" checkbox before clicking on the "create fork" button. This will allow you to also have the develop branch, since you will need the develop branch for the next steps).
+- Go to the develop branch of your fork.
+- Create a new branch from the develop branch and make your changes in the new branch you created.
+- Send a pull request pointing to the develop branch of the original training repo.
+- Wait for review of your changes from the repo admin(s).
```

Comment:
Some things here in the list were links and you did not format them as links that the reader can click on. Check the formattings... Links and any other type of formatting, and add them.